draft-foudil-securitytxt.md

```diff
@@ -177,6 +186,17 @@ We would like to thank the following researchers:
 (2016-06-10) Anna Richmond - A server configuration issue
 ~~~~~~~~~~
 
+### Canonical {#canonical}
+
+This directive indicates the canonical URI where the security.txt file is located,
+which is usually something like "example.com/.well-known/security.txt".
```

Comment:
Since the security.txt file should be served over HTTPS, I think it is safe to use `https://example.com/.well-known/security.txt` here.

Reply:
Updated with this change, plus the language changes we discussed